doc/sphinx/build-bastion-conf-help.pl

```diff
@@ -0,0 +1,117 @@
+#! /usr/bin/env perl
+# vim: set filetype=perl ts=4 sw=4 sts=4 et:
+use strict;
+use warnings;
+
+my %h;
+my @out;
+my $section;
+my %sections;
+my %sectiondesc;
+my @orderedsections;
+
+sub dumpdoc {
+    if (!%h) {
+        ;    # nothing to do
+    }
+    elsif (defined $h{param} && defined $h{default} && defined $h{desc} && defined $h{type}) {
+        die "attempting to dump data but section=$section" if !$section;
+        push @{$sections{$section}}, $h{param};
```

Comment:
This could be something like

```perl
push @{$sections{$section}},
    $h{param},
    ".. _bastion_conf_$h{param}:\n"
    "\n",
    "$h{param}\n",
    "*" x length($h{param}), "\n\n",
    ...
```